pyccel/codegen/printing/cwrappercode.py

```diff
@@ -518,7 +696,12 @@ def _print_Module(self, expr):
             static_funcs = expr.funcs
         function_signatures = '\n'.join('{};'.format(self.function_signature(f)) for f in static_funcs)
 
-        function_defs = '\n\n'.join(self._print(f) for f in expr.funcs)
+        #TODO it should be a better way to do this :
+        funcs = expr.interfaces
```

Comment:
Does this method create a copy of a list? If not are we in danger of modifying the `Module` contents here?

Reply:
non it doesn't create a copy,  And I don't see where is a danger as long as we don't modify it.

Reply:
Ok. Now the names have been changed it seems more obvious that this is a temporary variable that is not modified :+1: